cognite/client/_api/synthetic_time_series.py

```diff
@@ -79,7 +78,7 @@ def retrieve(
         datapoints.external_id = short_expression  # for dataframe readability
         while True:
             query["limit"] = min(limit, self._DPS_LIMIT)
-            resp = self._post(url_path=self._SYNTHETIC_RESOURCE_PATH + "/query", json={"items": [query]})
+            resp = self._post(url_path=self._RESOURCE_PATH + "/query", json={"items": [query]})
```

Comment:
api supports multiple, but we support only single expression per request?

Reply:
redone